src/app/core/auth/types/credentials.ts

```diff
@@ -2,3 +2,5 @@ export type Credentials = {
   email: string
   password: string
 }
+
+export type RegisterData = { name: string } & Credentials
```

Comment:
Why call it differently (Data), when we already have a name for this kind of information (Credential)? Be consistent in naming things